userops_ext.go

```diff
@@ -157,9 +157,12 @@ func (op *UserOperation) extractIntentJSON() (string, bool) {
 func ExtractJSONFromField(fieldData string) (string, bool) {
 	if fieldData != "" {
 		var intent pb.Intent
-		if err := protojson.Unmarshal([]byte(fieldData), &intent); err == nil {
-			return fieldData, true
+		err := protojson.Unmarshal([]byte(fieldData), &intent)
+		if err != nil {
+			fmt.Printf("Error unmarshalling JSON to Intent protobuf: %v\n", err)
```

Comment:
Until we add a logger,  error messages should target `stderr`, because docker and monitoring systems treat stderr stream differently for errors.

Reply:
its was locally added to console.log allong with debugger to catch the error